lib/matplotlib/backend_bases.py

```diff
@@ -2763,6 +2763,10 @@ def draw_rubberband(self, event, x0, y0, x1, y1):
         """Draw a rectangle rubberband to indicate zoom limits"""
         pass
 
+    def remove_rubberband(self):
```

Comment:
Other way around, the toolbase work is functionality that is just starting to be built out by @fariza and @OceanWolf